src/components/image/image.spec.tsx

```diff
@@ -29,36 +29,33 @@ describe("Image", () => {
   });
 
   describe("prop types", () => {
-    beforeEach(() => {
-      jest.spyOn(global.console, "error").mockImplementation(() => {});
-    });
+    const mockGlobal = jest
+      .spyOn(global.console, "error")
+      .mockImplementation(() => undefined);
 
     afterEach(() => {
-      global.console.error.mockReset();
+      mockGlobal.mockReset();
     });
 
     it("throws an error when `children` and `src` are passed", () => {
-      mount(
-        <Image src="foo.jpg" alt="foo">
-          foo
-        </Image>
-      );
-      expect(console.error).toHaveBeenCalled();
+      const errorMessage =
+        "The 'Image' component renders as an 'img' element when the 'src' prop is used and therefore does not accept children.";
+      expect(() => {
+        mount(
+          <Image src="foo.jpg" alt="foo">
+            foo
+          </Image>
+        );
+      }).toThrow(errorMessage);
     });
 
     it("throws an error when `src` is passed and no `alt` provided", () => {
-      mount(<Image src="foo.jpg" />);
-      expect(console.error).toHaveBeenCalled();
-    });
-
-    it("does not throw an error when `src` is passed and `alt` provided", () => {
```

Comment:
is there a good reason for removing this test? Why not something like `expect(() => { mount(<Image src="foo.jpg" alt="foo"/>); }).not.toThrow();`?

Reply:
Oh yes, of course. I withdraw the comment then :)